libsolidity/codegen/ir/IRGeneratorForStatements.cpp

```diff
@@ -2073,7 +2071,7 @@ void IRGeneratorForStatements::endVisit(IndexAccess const& _indexAccess)
 					IRVariable(_indexAccess.baseExpression()).commaSeparatedList() +
 					", " +
 					expressionAsType(*_indexAccess.indexExpression(), *TypeProvider::uint256()) +
-					")\n";
+					")";
```

Comment:
Considering `indexAccessFunctionCall` an expression, so moved this termination to the statement part below.